config/config.ini

```diff
@@ -59,7 +59,7 @@
 # Workspace One section is applicable only when Workspace One backend is enabled in the [main] section.
 
 # Uncomment to provide Workspace One token. Alternatively, use WORKSPACEONE_TOKEN variable
```

Comment:
```suggestion
# Uncomment to provide Workspace One token. Alternatively, use `token` variable
```

Reply:
Comment should be updated if the variable name is changed

Reply:
Cannot change the documentation here without changing the code. The environment variable WORKSPACE_TOKEN is still required by code.

Let me step back a bit and describe my thought process about naming things. I prefer to have the environment variables names to be fully qualified. So it is very clear what the environment variable configures. On the other hand our config file is structured, modularized and thus the variable in given config modules does not need to be fully qualified as it is clear that it relates to the given section/module.